proto/beacon/p2p/v1/generated.ssz.go

```diff
@@ -2663,7 +2663,9 @@ func (s *SyncCommittee) HashTreeRootWith(hh *ssz.Hasher) (err error) {
 				err = ssz.ErrBytesLength
 				return
 			}
-			hh.Append(i)
+			padded := make([]byte, 64)
+			copy(padded[0:48], i[0:48])
```

Comment:
Curious here, why does it make a byte slice of length 64, but only assign 48 bytes of it?

Reply:
zero padding to the next chunk (chunks are 64 bytes) -- create a slice of 64 zeros, overwrite the first 48.